docs/product-user-guide/src/main/asciidoc/interacting-with-processes-delegating-claiming-releasing-tasks-proc.adoc

```diff
@@ -1,19 +1,19 @@
 [id='interacting-with-processes-delegating-claiming-releasing-tasks-proc']
 = Delegating, claiming, and releasing tasks
 
-Once tasks are created in {CENTRAL}, you can delegate them to others, claim released tasks, and release your tasks for others to claim. Note that every user may not have permission for delegating, claiming, or releasing tasks.
+After tasks are created in {CENTRAL}, you can delegate them to others, claim released tasks, and release your tasks for others to claim. Only users with admin or process-admin permissions can delegate, claim, or release tasks.
```

Comment:
This statement is incorrect:
_Only users with admin or process-admin permissions can delegate, claim, or release tasks._
If a user can see the task, the user can perform all these actions, no matter the role.

Reply:
change ack